app/utils/helpers.ts

```diff
@@ -95,6 +95,19 @@ export function formatDate(dateString: string | Date, locale = 'de-DE') {
   const month = date.toLocaleString(locale, { month: 'long' });
   const year = date.toLocaleString(locale, { year: 'numeric' });
 
+  return `${day}. ${month} ${year}`;
+}
+
+export function formatDateWithTimestamp(dateString: string | Date, locale = 'de-DE') {
```

Comment:
below this, there is a function `formatTimestamp()`, which seems to be a duplciate of `formatDate()`, can you please make sure it's not left there unnecessarily?